app/services/analytics_events.rb

```diff
@@ -2312,32 +2312,37 @@ def user_registration_email_confirmation(
     )
   end
 
-  # Tracks exceptions that are raised when running GetUspsProofingResultsJob
-  # @param [String] reason why was the exception raised?
+  # Tracks if when a user is enrolled in in-person proofing
+  # @param [String] context
+  def idv_in_person_usps_request_enroll(context:, **extra)
+    track_event(
+      'USPS IPPaaS enrollment',
+      context: context,
+      **extra,
+    )
+  end
+
+  # Tracks if USPS in-person proofing enrollment request fails
+  # @param [String] context
+  # @param [String] reason
   # @param [String] enrollment_id
   # @param [String] exception_class
   # @param [String] exception_message
-  def idv_in_person_usps_proofing_results_job_exception(
-    reason:, enrollment_id:, exception_class: nil, exception_message: nil, **extra
+  def idv_in_person_usps_request_enroll_exception(
+    context:,
+    reason:,
+    enrollment_id:,
+    exception_class:,
+    exception_message:,
+    **extra
   )
     track_event(
-      'GetUspsProofingResultsJob: Exception raised',
-      reason: reason,
+      'USPS IPPaaS enrollment failed',
+      context: context,
       enrollment_id: enrollment_id,
       exception_class: exception_class,
       exception_message: exception_message,
-      **extra,
-    )
-  end
-
-  # Tracks individual enrollments that fail during GetUspsProofingResultsJob
-  # @param [String] reason why did this enrollment fail?
-  # @param [String] enrollment_id
-  def idv_in_person_usps_proofing_results_job_enrollment_failure(reason:, enrollment_id:, **extra)
```

Comment:
Oh, I think this was just me accidentally overwriting instead of copying and modifying. I put up with WIP PR to let the CI run tests but then had to switch tasks -- I just hadn't noticed this mistake yet

Reply:
Fixed, thanks